Makefile

```diff
@@ -166,9 +166,7 @@ test-k8s:
 	cp $(DEPLOY_DIR)/*.yaml $$TMP_DIR/; \
        sed -i 's/emulatedMode: .*/emulatedMode: "$(EMULATED_MODE)"/' $$TMP_DIR/03_instaslice_operator.cr.yaml; \
        env IMAGE_REGISTRY=$(IMAGE_REGISTRY) IMAGE_TAG=$(IMAGE_TAG) envsubst < $(DEPLOY_DIR)/04_deployment.yaml > $$TMP_DIR/04_deployment.yaml; \
-       env IMAGE_REGISTRY=$(IMAGE_REGISTRY) IMAGE_TAG=$(IMAGE_TAG) envsubst < $(DEPLOY_DIR)/05_scheduler_deployment.yaml > $$TMP_DIR/05_scheduler_deployment.yaml; \
        kubectl apply -f $$TMP_DIR/; \
```

Comment:
I think this has to be `kubectl apply -f $$TMP_DIR/`
Resolved this as part of https://github.com/openshift/instaslice-operator/pull/669